src/XperienceCommunity.HCaptcha/XperienceCommunity.HCaptcha.csproj

```diff
@@ -17,7 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Kentico.Xperience.AspNetCore.WebApp" Version="[13.0.66, 13.1.0)" />
+    <PackageReference Include="Kentico.Xperience.AspNetCore.WebApp" Version="13.0.82" />
```

Comment:
Was there a reason you changed the hotfix version?

Reply:
Sorry on the version change, it was just to match the solution so i could test, but looking at it probably would have worked without.

Reply:
The `packages.lock.json` will need regenerated as well by doing a build of the project (or you can revert it to what was there originally).

This file works the same as the `package-lock.json` does with npm/node.